spec/components/preview_component/view_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe PreviewComponent::View, type: :component do
+  it "shows in preview_draft" do
+    mode = Mode.new('preview-draft')
+    render_inline(described_class.new(mode: mode))
+    expect(page).to have_selector(".govuk-phase-banner")
+    expect(page).to have_content("Draft preview")
+    expect(page).to have_content("You're previewing the draft version of this form")
```

Comment:
Change to curly quotes is causing the test failure here:

```suggestion
    expect(page).to have_content("Draft preview")
    expect(page).to have_content("You’re previewing the draft version of this form")
```
or call the translation directly:
```suggestion
    expect(page).to have_content(I18n.t("mode.phase_banner_tag_preview-draft"))
    expect(page).to have_content(I18n.t("mode.phase_banner_text_preview-draft"))
```